src/engraving/dom/chord.h

```diff
@@ -212,7 +212,7 @@ class Chord final : public ChordRest
     Note* selectedNote() const;
 
     PointF pagePos() const override;        ///< position in page coordinates
-    void cmdUpdateNotes(AccidentalState*);
+    void cmdUpdateNotes(AccidentalState*, size_t staffIdx);
```

Comment:
Better to use `staff_idx_t` (which is just an alias for size_t but it's good for clarity and consistency)